app/poros/event.rb

```diff
@@ -1,15 +1,22 @@
 class Event
-  attr_reader :id, :user_id, :gym_membership_id, :date_time, :activity
+  attr_reader :id, :user_id, :invitee_name, :gym_membership_id, :date_time, :activity
 
   def initialize(data)
     @id                = data[:id]
     @user_id           = data[:attributes][:user_id]
+    @invitee_name      = data[:relationships][:user][:meta][:full_name]
     @gym_membership_id = data[:attributes][:gym_membership_id]
-    @date_time         = format_date(data[:attributes][:date_time])
+    @date_time         = data[:attributes][:date_time]
     @activity          = data[:attributes][:activity]
   end
 
-  def format_date(date)
-    date.to_datetime.strftime('%A, %b %d, %Y %l:%M%P')
+  # TODO: Update method to handle different timezones
+  def format_date
+    DateTime.strptime(date_time, '%Y-%m-%dT%H:%M:%S.000').strftime('%A, %b %d, %Y %l:%M%P')
+  end
+
+  # TODO: Update method to handle different timezones
+  def format_date_short
+    DateTime.strptime(date_time, '%Y-%m-%dT%H:%M:%S.000').strftime('%a %m/%d/%y %l:%M%P')
   end
```

Comment:
We'll need to revisit this and how we are formatting the dates.  Right now, the methods assume that the incoming dates are in 000Z timezone.

Reply:
I updated this in a later commit here